tests/dynamic_data/publication_test.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from .. import dynamic_data as data
+
+
+def run_action(workspace, publ_type, publication, action):
+    params = {
+        'publication_type': publ_type,
+        'workspace': workspace,
+        'name': publication,
+    }
+    params.update(action.params)
+    action.method(**params)
+
+
+@pytest.mark.parametrize('workspace, publ_type, publication', data.PUBLICATIONS)
+@pytest.mark.usefixtures('liferay_mock', 'ensure_layman')
+def test_publication_chain(workspace, publ_type, publication):
+    for step in data.PUBLICATIONS[(workspace, publ_type, publication)]:
```

Comment:
Use tuple `publication` instead of three variables "workspace, publ_type, publication" if possible